src/utils.ts

```diff
@@ -130,117 +126,97 @@ export function computeDustAmount(currency: ICrypto, txSize: number) {
   return dustAmount;
 }
 
-export function isValidAddress(address: string) {
+/**
+ * Temporarily copied from bitcoinjs-lib master branch (as of 2021-09-02,
+ * commit 7b753caad6a5bf13d40ffb6ae28c2b00f7f5f585) so that we can make use of the
+ * updated bech32 lib version 2.0.0 that supports bech32m. bitcoinjs-lib version 5.2.0
+ * as currently used by wallet-btc uses an older version of bech32 that lacks bech32m support.
+ *
+ * When a new version of bitcoinjs-lib that supports bech32m is released this function can
+ * be removed and calls should be directed to bitcoinjs-lib instead. Our direct dependency
+ * on bech32 lib should also be removed.
+ */
```

Comment:
Thanks for documenting this :+1: can you please just add a `TODO` in your comment to flag it for later.